packages/ui/src/components/Button/Button.stories.tsx

```diff
@@ -54,6 +54,7 @@ const Template: Story = () => {
         data-h2-display="base(grid)"
         data-h2-grid-template-columns="base(1fr 1fr 1fr 1fr)"
         data-h2-text-align="base(center)"
+        data-h2-color="base(red)"
```

Comment:
While I test this, should we remove these?

Reply:
Honestly, I kind of like keeping them there just as a safety... Things will be real obvious if they stop working for whatever reason.